build.sh

```diff
@@ -18,4 +18,10 @@ sed -e "/#INSERT_PYTHON_CODE_HERE/r ./src/aws_cli_mfa.py" -e "s///" ./src/$AWS_C
     sudo chmod +x /usr/local/bin/aws-cli-mfa &&
     cp /usr/local/bin/aws-cli-mfa ./bin/aws-cli-mfa-$AWS_CLI_MFA_SHELL
 
+sed 's/"/\\\"/g' ./src/aws_cli_mfa.py | sed -e "/#INSERT_PYTHON_CODE_HERE/r /dev/stdin" -e "s///" ./src/mfa.fish |
```

Comment:
instead of the sed, perhaps you should use fish's `string escape`? https://fishshell.com/docs/current/cmds/string.html

Reply:
I like this idea.  I wasn't sure if it was a good idea to mix fish code in the same file as bash/zsh/etc, but these clearly are meant to be used individually.

Reply:
`string escape` seems aimed at sanitizing fish script, urls and variables.  I ran a few tests, but wasn't able to get it to do the thing we need, `string replace --all '"' '\\"' < input_file` works great though.  I changed it in both the tests file and the `build.sh`